CHANGELOG.md

```diff
@@ -149,10 +95,50 @@ Updated the linter to `0.1.89`, which includes the following changes:
   This change only affects implementers of the `RegExp` interface; current
   code using Dart regular expressions will not be affected.
 
-#### `dart:isolate`
+### Language
 
-* **Breaking change:** The `await for` allowed `null` as a stream due to a bug
-  in `StreamIterator` class. This bug has now been fixed.
+*   **Breaking change:** Covariance of type variables used in super-interfaces
+    is now enforced (issue [35097][]).  For example, the following code was
+    previously accepted and will now be rejected:
+
+```dart
+class A<X> {};
+class B<X> extends A<void Function(X)> {};
+```
+
+* The identifier `async` can now be used in asynchronous and generator
+  functions.
+
+[35097]: https://github.com/dart-lang/sdk/issues/35097
+
+### Tools
+
+#### Linter
+
+The Linter was updated to `0.1.91`, which includes the following changes:
+
+* fixed missed cases in `prefer_const_constructors`
+* fixed `prefer_initializing_formals` to no longer suggest API breaking changes
+* updated `omit_local_variable_types` to allow explicit `dynamic`s
+* (internal) migration from deprecated analyzer APIs
```

Comment:
I don't think we need to expose internal linter changes in the SDK changelog.

Reply:
Updated.